.github/workflows/deploy.yml

```diff
@@ -22,4 +22,4 @@ jobs:
       
     - name: Trigger Vercel Deploy Hook
       run: |
-        curl -X POST ${{ secrets.VERCEL_DEPLOY_HOOK_URL }}?buildCache=${{ inputs.buildWithCache }}
+        curl -X POST ${{ secrets.VERCEL_DEPLOY_HOOK_URL }}?buildCache=${{ inputs.buildWithCache || false }}
```

Comment:
The `undefined` error is likely because Action/Repo secrets from the old repo aren't present 
 
<img width="881" alt="image" src="https://github.com/bigcommerce/docs/assets/29524703/e3666bf4-22e5-40d7-80a4-a4dce77d99fc">